src/Umbraco.Web/Compose/BlockEditorComponent.cs

```diff
@@ -116,8 +117,23 @@ private void RecursePropertyValues(IEnumerable<BlockItemData> blockData, Func<Gu
                             // this gets a little ugly because there could be some other complex editor that contains another block editor
                             // and since we would have no idea how to parse that, all we can do is try JSON Path to find another block editor
                             // of our type
-                            var json = JToken.Parse(asString);
-                            if (ProcessJToken(json, createGuid, out var result))
+                            JToken json = null;
+                            try
+                            {
+                                json = JToken.Parse(asString);
+                            }
+                            catch (Exception e)
+                            {
+                                // See issue https://github.com/umbraco/Umbraco-CMS/issues/10879
+                                // We are detecting JSON data by seeing if a string is surrounded by [] or {}
+                                // If people enter text like [PLACEHOLDER] JToken  parsing fails, it's safe to ignore though
+                                // Logging this just in case in the future we find values that are not safe to ignore
+                                Current.Logger.Warn<BlockEditorComponent>(
```

Comment:
I think you should inject the `ILogger` instead of using `Current`

Reply:
@bergmania  Yeah I tried that but it's a breaking change 😢 
Unit tests then need to provide an `ILogger` as well and since it's a public class, people could be using it.

Reply:
I would still add a default ctor, just injection `Current.Logger` into the other ctor.. That's the way we normally handle such things..

Reply:
```
        private readonly ILogger _logger;

        [Obsolete("Use the ctor injecting dependencies.")]
        public BlockEditorComponent() : this(Current.Logger)
        {
        }

        public BlockEditorComponent(ILogger logger)
        {
            _logger = logger;
        }
```

And then use `_logger` in the code

Reply:
Ah.. that makes sense! Yes, that should be fine, I'll update!

Reply:
That's now fixed! 👍